packages/react-composites/tests/browser/call/CallComposite.test.ts

```diff
@@ -37,6 +37,7 @@ test.describe('Call Composite E2E Tests', () => {
       await stubLocalCameraName(page);
       await page.waitForSelector(dataUiId('call-composite-device-settings'));
       await page.waitForSelector(dataUiId('call-composite-local-preview'));
+      await pages[idx].waitForSelector(`${dataUiId('call-composite-start-call-button')}[data-is-focusable="true"]`);
```

Comment:
Added this to fix issue I encountered where actual snapshot was with info on device half updated 
![image](https://user-images.githubusercontent.com/82416644/131907470-c3ce9855-6c22-4d3c-ad45-6c1067f703ad.png)
So waiting for the button to be enabled , so we have a stable state for comparison

Reply:
pages[idx] could just be page